app/screens/send-bitcoin-screen/destination-information.tsx

```diff
@@ -6,21 +6,22 @@ import { useAppConfig } from "@app/hooks"
 import React from "react"
 import { Text, View } from "react-native"
 import EStyleSheet from "react-native-extended-stylesheet"
-import { SendBitcoinDestinationState } from "./send-bitcoin-reducer"
+import { DestinationState, SendBitcoinDestinationState } from "./send-bitcoin-reducer"
 import { IntraledgerPaymentDestination } from "@galoymoney/client/dist/parsing-v2"
+import { InvalidDestinationReason } from "./payment-destination/index.types"
 
 const toLnAddress = (handle: string, lnDomain: string) => {
   return `${handle}@${lnDomain}`
 }
 
 const destinationStateToInformation = (
-  destinationState: SendBitcoinDestinationState,
+  sendBitcoinReducerState: SendBitcoinDestinationState,
   translate: TranslationFunctions,
```

Comment:
you could just pass `translate.SendBitcoinDestinationScreen`. I think otherwise we typically call the translate function `TT` in the other part of the app (assuming this is the same thing). we should keep the same function name here

Reply:
I think this should be handled in when creating the fix for issue #1089